client/my-sites/stats/site.jsx

```diff
@@ -144,8 +147,29 @@ class StatsSite extends Component {
 		this.props.recordTracksEvent( 'calypso_stats_parsely_banner_click' );
 	};
 
+	buildPrivateSiteBanner() {
```

Comment:
Nit: it may be better to call this `renderPrivateSiteBanner()` -- the `render` prefix makes the purpose a bit clearer (even if the code is technically building the object).

Reply:
Cool! My way of thinking to place that is that when I not render it directly into the HTLM section of the JSX/TSX I tend to prefix it as build, when I build and render inline then I tend to prefer it render. So anyway as I've made it now inline I've prefixed it to render